### PR TITLE
Update appropriate field for the form control

### PR DESCRIPTION
**Pull request by ajanec01**
Re-wording the definition of [Appropriate field for the form control](https://act-rules.github.io/rules/73f2c2#appropriate-field-for-the-form-control) so that it references required control groups/ input states.

Closes issue #1366

### CODE REVIEW

File: pages/glossary/appropriate-field-for-the-form-control.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ input_aspects:
   - DOM tree
 ---
 
-The field name listed in the autocomplete fields table from the HTML 5.2 specification is used in the `autocomplete` attribute of an element: [https://html.spec.whatwg.org/#inappropriate-for-the-control](https://html.spec.whatwg.org/#inappropriate-for-the-control)
+The field name listed in the autocomplete fields table from the HTML 5.2 specification is appropriate if the form control it applies to belongs to the specified control group. [https://html.spec.whatwg.org/#inappropriate-for-the-control](https://html.spec.whatwg.org/#inappropriate-for-the-control).
```
Comment (carlosapaduarte) [inline pages/glossary/appropriate-field-for-the-form-control.md:10]:
```suggestion
The field name of the form control is appropriate if it is listed in the autocomplete fields table from the HTML 5.2 specification as applying to the specified control group. [https://html.spec.whatwg.org/#inappropriate-for-the-control](https://html.spec.whatwg.org/#inappropriate-for-the-control).
```

File: _rules/aria-required-owned-element-bc4a75.md [hunk 2]
```diff
@@ -65,6 +57,12 @@ If the [explicit semantic role][] on the target element is incorrectly used, and
 - [Required Owned Element](https://www.w3.org/TR/wai-aria-1.1/#mustContain)
 - [Owned Element](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element)
 
+HTML elements with an [implicit semantic role][] corresponding with [explicit semantic role][] are not tested in this rule because some of these elements have additional requirements. For example, an element that has an implicit semantic role that is identical to its explicit semantic role is a `ul` element that has `role="list"`. However, there is no native HTML element to "group" `li` elements in a `ul` element.
+
+The unresolved issues with how [Digital Publishing WAI-ARIA Module][] (DPUB ARIA 1.1) uses role inheritance to define the [required owned element][] is limiting the scope of this rule to [WAI-ARIA 1.1][] roles only. It is because of the deviations between DPUB ARIA 1.1 and [WAI-ARIA 1.1][] models. The [WAI-ARIA Graphics Module][] does not include any [required owned element][].
```
Comment (carlosapaduarte) [inline _rules/aria-required-owned-element-bc4a75.md]:
```suggestion
The unresolved issues with how [Digital Publishing WAI-ARIA Module][] (DPUB ARIA 1.1) uses role inheritance to define the [required owned element][] limit the scope of this rule to [WAI-ARIA 1.1][] roles only. The [WAI-ARIA Graphics Module][] does not include any [required owned element][].
```

Comment (ajanec01) [reply _rules/aria-required-owned-element-bc4a75.md]: I didn't mean to change the aria-required-own-element in this PR. I messed up a biut by creating a branch from #1365 but I will add those changes to it. Thanks!

File: pages/glossary/appropriate-field-for-the-form-control.md [hunk 3]
```diff
@@ -7,4 +7,4 @@ input_aspects:
   - DOM tree
 ---
 
-The field name listed in the autocomplete fields table from the HTML 5.2 specification is used in the `autocomplete` attribute of an element: [https://html.spec.whatwg.org/#inappropriate-for-the-control](https://html.spec.whatwg.org/#inappropriate-for-the-control)
+The field name of the form control is appropriate if it is listed in the autocomplete fields table from the HTML 5.2 specification as applying to the specified control group. [https://html.spec.whatwg.org/#inappropriate-for-the-control](https://html.spec.whatwg.org/#inappropriate-for-the-control).
```
Comment (Jym77) [inline pages/glossary/appropriate-field-for-the-form-control.md:10]:
```suggestion
The field name of the form control is _appropriate_ if it is listed in the autocomplete fields table from the HTML 5.2 specification as applying to the specified control group. [https://html.spec.whatwg.org/#inappropriate-for-the-control](https://html.spec.whatwg.org/#inappropriate-for-the-control).
```
Taking the opportunity of the new formulation to emphasise the defined term 😄

**Pull request merged**